src/utils/SpokeUtils.ts

```diff
@@ -455,6 +455,7 @@ export async function findFillEvent(
   const fill = {
     ...spreadEventWithBlockNumber(event),
     destinationChainId,
+    messageHash: getMessageHash(event.args.message),
   } as FillWithBlock;
```

Comment:
How does this not get caught by typescript compile time?? ugh

Reply:
It's because of the nasty type assertion on line 459. It's a real anti-pattern to do this and we only get away with it because it's in a low-level function that rarely changes; otherwise it'd blow up regularly.